src/main/java/core/basesyntax/GenerateArray.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class GenerateArray {
+    private static Colors[] color = Colors.values();
+
+    public static void main(String[] args) {
+        Random rnd = new Random();
+        int x = rnd.nextInt(10);
+        Object [] randomArray = new Object[x];
+        for (int i = 0; i < x; i++) {
+            randomArray[i] = random();
+        }
+        for (Object print: randomArray) {
+            System.out.println(print.toString());
+        }
+    }
+
+    private static Object random() {
+        Random figures = new Random();
+        int y = figures.nextInt(5);
```

Comment:
- `y` - is a very strange variable name 
- why do we use 5? not 6 or 9? let's create a constant for this variable - https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names